changelogs/unreleased/gh-7700-interval-arithmetic-across-dst.md

```diff
@@ -0,0 +1,20 @@
+## bugfix/datetime
+
+* fix interval arithmetic for boundaries crossing DST
```

Comment:
Nit: should be "Fixed interval arithmetic for boundaries crossing DST (gh-7700)."

Also please align both changelogs so that the paragraphs below are part of the bullet (indent them by 2 spaces)

Reply:
Fixed in both commits. Thanks!